packages/terra-table/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "terra-table",
   "main": "lib/index.js",
-  "version": "5.2.0",
+  "version": "5.3.0",
```

Comment:
It looks like it has only fix, shouldn't it be "5.2.1"?

Reply:
You're right, thanks for catching that! Fixed here: 3f7b97640071c32145559157f750396ad2285888